MaterialLib/Fluid/Viscosity/createViscosityModel.cpp

```diff
@@ -32,7 +32,7 @@ namespace Fluid
 static std::unique_ptr<FluidProperty> createLinearPressureDependentViscosity(
     BaseLib::ConfigTree const& config)
 {
-    //! \ogs_file_param{material__fluid__viscosity__type}
+    //! \ogs_file_param{material__fluid__viscosity__LinearPressure}
```

Comment:
In ogs_file_param{xxx} the xxx refers to the first argument to the checkConfigParameter() call. So in this case the original version with '__type' is correct. Only the subsequent references in ogs_file_param include the specific case , e.g. LinearPressure.